README.md

```diff
@@ -24,80 +23,119 @@
 
 ## :star: Introduction
-B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital products across multiple platforms and brands.
 
-B01LER provides pre-built components that exist in design in the form of a [component library in Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1) (the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the organization can download the file [here](https://boilerds.com/boilerds/B01LER-v1-alpha%20release.fig)), as well as in code in a repository (the one you have opened right now), which is also [displayed in the web using Storybook](https://b01ler.onrender.com/). Both the components in design and in code are connected via a token system.
+B01LER is a fully customisable design system (DS) that enables you to easily create cohesive and consistent digital
+products across multiple platforms and brands.
+
+B01LER provides pre-built components that exist in design in the form of a
+[component library in Figma](https://www.figma.com/file/zPHgVJFeLAhdM4MdOymhPb/B01LER--%5Bv0.1---alpha-release%5D?type=design&node-id=8941%3A1818&mode=design&t=owVepZzAwdrehnmL-1)
+(the link to the Figma file above is exclusively for members of the Accenture organization, while those outside the
+organization can download the file [here](https://boilerds.com/boilerds/B01LER-v1-alpha%20release.fig)), as well as in
+code in a repository (the one you have opened right now), which is also
+[displayed in the web using Storybook](https://b01ler.onrender.com/). Both the components in design and in code are
+connected via a token system.
 
-A token system is a collection of values which represent the visual properties of a design system, such as typography, color and spacing.
+A token system is a collection of values which represent the visual properties of a design system, such as typography,
+color and spacing.
 
-Utilizing tokens, B01LER is simultaneously consistent and flexible. By changing the tokens, you have the power to change the appearance of the whole design, including all the components and all the digital products build with it. In Figma and in code.
+Utilizing tokens, B01LER is simultaneously consistent and flexible. By changing the tokens, you have the power to change
+the appearance of the whole design, including all the components and all the digital products build with it. In Figma
+and in code.
 
-Overall, B01LER is a powerful tool that helps you work smarter, not harder. By streamlining the design and development processes B01LER empowers you to create better products faster.
+Overall, B01LER is a powerful tool that helps you work smarter, not harder. By streamlining the design and development
+processes B01LER empowers you to create better products faster.
 
 ### Benefits of B01LER
+
```

Comment:
Every file has now a new line added after every headlines. This usually happens because of auto-format while saving in editor. Anyway were these added new lines intentional?

Reply:
This was not intended. Would you recommend to remove these again? I do not have a preference.

Reply:
The previous state was better i.e. without new line and sentences not wrapped prematurely